### PR TITLE
Add a configuration setting to disable typename injection

### PR DESCRIPTION
**Pull request by msullivan**
This will be useful for me doing development. If there is a way to
make the setting undocumented, it might be worth doing that, since it
seems fairly marginal for normal usage.

### CODE REVIEW

File: src/commands/parser.rs [hunk 1]
```diff
@@ -156,6 +156,8 @@ pub enum Setting {
     Limit(Limit),
     /// Set output format.
     OutputFormat(OutputFormat),
+    /// Whether to inject typenames in default output mode
```
Comment (elprans) [inline src/commands/parser.rs:159]:
```suggestion
    /// Resolve object type names in default output mode
```

**Comment by elprans** [comment]
> If there is a way to make the setting undocumented

We could add a "devmode" feature to the CLI crate and enable it when building the CLI from `setup.py`.

**Review by elprans — APPROVED** [review]

File: src/commands/parser.rs (hunk 1 above)
Comment (msullivan) [reply src/commands/parser.rs:159]: How about "Display object type names"?

**Comment by tailhook** [comment]
Technically we have `dev_mode` feature and we build with it from `setup.py`.

**Pull request merged**